src/components/maps/CameraViewer.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Actions } from 'jumpstate';
+
+import Anchor from 'grommet/components/Anchor';
+import Box from 'grommet/components/Box';
+import Button from 'grommet/components/Button';
+import Label from 'grommet/components/Label';
+import Layer from 'grommet/components/Layer';
+import List from 'grommet/components/List';
+import ListItem from 'grommet/components/ListItem';
+import Notification from 'grommet/components/Notification';
+import Tile from 'grommet/components/Tile';
+import Tiles from 'grommet/components/Tiles';
+import FormNextIcon from 'grommet/components/icons/base/FormNext';
+import FormPreviousIcon from 'grommet/components/icons/base/FormPrevious';
+import SpinningIcon from 'grommet/components/icons/Spinning';
+
+import {
+  MAPEXPLORER_INITIAL_STATE, MAPEXPLORER_PROPTYPES,
+  MAPEXPLORER_CAMERA_STATUS,
+} from '../../ducks/mapexplorer';
+
+const ITEMS_PER_PAGE = 6;
+const THUMBNAIL_SERVER_URL = 'https://thumbnails.zooniverse.org/';
+const THUMBNAIL_WIDTH = 320;
+const THUMBNAIL_HEIGHT = 200;
+
+class CameraViewer extends React.Component {
+  constructor(props) {
+    super(props);
+  
+    this.renderData = this.renderData.bind(this);
+    this.renderMetadata = this.renderMetadata.bind(this);
+    this.renderDataPaging = this.renderDataPaging.bind(this);
+    this.changeDataPaging = this.changeDataPaging.bind(this);
+  
+    this.state = {
+      page: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      page: 0,
+    });
+  }
+  
+  render() {
+    if (!this.props.mapConfig) return null;
+    
+    return (
+      <Layer className="camera-viewer">
```

Comment:
I originally removed the `closer` code because I couldn't get the Close button styled the way I wanted (I actually cribbed the `<Layer>` code from your CreateClassroomForm - thanks, btw!) but this might go back to my comments about 'fighting with the Grommet framework too much'. I'll try re-enabling the standard `closer` code and see if I can get the style working - it's better that the EAPIFE shares as consistent a UI as possible, after all.